sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Release History
 
+## 12.0.0-beta.5 (2020-11-13)
+
+### New Features
+
+* Support for replaceable HTTP transport layer
+
+### Breaking Changes
+
+* API return types are moved to `Models` namespace, other stuff in protocol layer is moved to `Details` namespace
+* Move XML serializer and deserializer to `Details` namespace
+* Remove `BlockBlobClientOptions`, `AppendBlobClientOptions` and `PageBlobClientOptions`, use `BlobClientOptions` instead
+* Rename `BlobSasBuilder::ToSasQueryParameters` to `BlobSasBuilder::GenerateSasToken`
+
+### Other Changes and Improvements
```

Comment:
No bug fixes section?

I recommend going through the .hpp file changes from all commits between the last release and this one to make sure all user-facing/relevant changes are highlighted.

Reply:
I already did